iocore/net/SSLUtils.cc

```diff
@@ -2007,7 +2007,10 @@ SSLParseCertificateConfiguration(const SSLConfigParams *params, SSLCertLookup *l
         if (ssl_extract_certificate(&line_info, sslMultiCertSettings)) {
           // There must be a certificate specified unless the tunnel action is set
           if (sslMultiCertSettings.cert || sslMultiCertSettings.opt != SSLCertContext::OPT_TUNNEL) {
-            ssl_store_ssl_context(params, lookup, &sslMultiCertSettings);
+            if (ssl_store_ssl_context(params, lookup, &sslMultiCertSettings) == nullptr) {
+              Error("failed to load SSL server contexts");
+              return false;
+            }
           } else {
```

Comment:
Yeh, it's not used, but the point is that we stop parsing the configuration at this point, which leaves lookup->is_valid == false (correctly). I believe in the old version, the status of the lookup is the status of the *last* certificate in the configuration.